OpenRA.Mods.Common/Traits/World/Locomotor.cs

```diff
@@ -461,12 +463,15 @@ void UpdateCellBlocking(CPos cell)
 								actorBits = actorBits.Union(crushingBits);
 								cellBlocking |= CellBlocking.Crushable;
 							}
-
-						if (isMoving)
-							actorBits = actorBits.Except(actor.Owner.AllyMask);
 					}
 					else
-						actorBits = isMoving ? actor.Owner.EnemyMask : allPlayerMask;
+						actorBits = allPlayerMask;
+
+					if (isMoving)
+					{
+						actorBits = actorBits.Except(actor.Owner.AllyMask);
+						cellBlocking |= CellBlocking.IsMoving;
```

Comment:
When _any_ actor in the cell is _not_ moving `IsMoving` should not be flagged. Might be easier to just reverse the meaning.